app.js

```diff
@@ -12,7 +23,58 @@ app.set('views', __dirname + '/views');
 app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
+app.get("/", (req, res) => {
+    res.render("index")
+})
+//res.render(data.body.artists.items[0].images)
+app.get("/artist-search", (req, res) => {
+    console.log("Artist came from browser - request")
+    console.log(req.query.artist)
+
+    //creating an object
+
+
+    spotifyApi.searchArtists(req.query.artist)
+        //.then(data => {console.log(data.body.artists.items)})
+        .then((data) => {
+
+            const artistImage = data.body.artists.items[0].images[0].url;
+            const artistName = data.body.artists.items[0].name;
+            const artistId = data.body.artists.items[0].id;
+            // console.log(artistObject)
+            res.render("artist-search-result", { artistImage: artistImage, artistName: artistName, artistId: artistId })
+        })
+
+        //res.render("artist-search-result",{artist:data.body.artists.items})})
+        .catch(err => console.log(err))
+})
+
+
+
+app.get("/albums/:artistId/", (req, res, next) => {
+    // .getArtistAlbums() code goes here
+
```

Comment:
A good practice and for easy to read its to save the params  in variables first and then pass it something like
```
const artistId =req.params.artistId
  spotifyApi
    .getArtistAlbums( artistId )
    ```